akka-docs/src/main/paradox/typed/routers.md

```diff
@@ -57,10 +57,15 @@ The routees, however, are spawned by the router.
 Therefore, the `PoolRouter` has a property to configure the `Props` of its routees:
 
 Scala
-:  @@snip [RouterSpec.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala) { #pool-dispatcher }
+:  @@snip [RouterSpec.scala](/akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala) { #broadcast }
 
 Java
-:  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #pool-dispatcher }
+:  @@snip [RouterTest.java](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/RouterTest.java) { #broadcast }
+
+### Broadcasting a message to all routees
+
+Pool routers can be configured to identify messages intended to be broad-casted to all routees.
+Therefore, the `PoolRouter` has a property to configure its `broadcastPredicate`: 
```

Comment:
I think I'd prefer that rather than build on top of the pool samples making it longer/using features not yet described in the doc page, only having a snippet showing setting the predicate would be enough (and either hide the broadcast message type from the other sample or simply using some dummy protocol only for this sample).